.gitignore

```diff
@@ -41,3 +41,5 @@ release/*
 
 node_modules/*
 
+package-lock.json

```

Comment:
aren't we supposed to source control this?
we do in every other repo

Reply:
you are correct - I left it out because previously it wasn't in source control, but I agree we should.  I will remove this and add my package lock file